### ISSUE TITLE
JSON Schema date-time to support date-time  = full-date "T" full-time

### ISSUE THREAD

**Issue by SureshNatarajan** [dschnelldavis/angular2-json-schema-form#134]
## Issue type
I'm submitting a (check one):
```
[X ] Bug report
[ ] Feature request
[ ] Regression (something that used to work, but stopped working in a newer version)
[ ] Support request
[ ] Documentation issue or request
```

## Prerequisites
Before posting, make sure you do the following (check all):
```
[X ] Confirm you are using the latest versions of all necessary packages (or if not, explain why not)
[ X] Search GitHub for a similar issue or PR
[ X] If submitting a Support request, also search [Stack Overflow][stack-overflow] for similar issues
Note: Please cross-post GitHub support requests to [Stack Overflow][stack-overflow], and include a link in your GitHub issue to your Stack Overflow question. We do currently respond to support requests on GitHub, but we eventually expect to stop, and will then refer all support questions exclusively to Stack Overflow.
```

## Current behavior
I am currently using a date time format type in my JSON schema. After form gets rendered, the HTML5 date picker is shown. When I try to pick the date it results in an error.

Sample JSON Schema:

`{
  "type": "object",
  "title": "Date-Time-Required-Field-Check",
  "required": [
    "startdate"
  ],
  "properties": {
    "url": {
      "type": "string",
      "format": "uri"
    },
    "text": {
      "type": "string"
    },
    "startdate": {
      "type": "string",
      "format": "date-time"
    }
  }
}`

**Error displayed is : Must be a date-time formatted like "2000-12-31" or "2000-03-14T01:59.265"**

## Expected behavior
I would like to get the date time. Doesn't matter what the format is, but want to know how to fix this issue.

## IMPORTANT: How can we reproduce your problem?
I have given the sample JSON schema that I am using. It can be easily reproduced in the https://angular2-json-schema-form.firebaseapp.com/

## Environment
```
OS name & version: Any
Browser name & version: Any
Angular version: 4
Angular JSON Schema Form version(s): latest
Other relevant software or packages:
```
## Any other relevant information

**Comment by SureshNatarajan** [comment]
Hi @dschnelldavis,

Is there an alternate way to accept the date with time ?

**Comment by dschnelldavis** [comment]
Thanks for the good example JSON Schema.

This has been fixed as of version 0.6.0-alpha.3.

**Comment by SureshNatarajan** [comment]
Thankyou @dschnelldavis for fixing it quickly.
